modules/builders-virtual-environment.adoc

```diff
@@ -91,7 +91,7 @@ oc sa get-token -n virtual-builders quay-builder
 +
 [source,terminal]
 ----
-$ oc create token quay-builder -n virtual-builders
+$ oc create token quay-builder -n virtual-builders —duration
```

Comment:
```suggestion
$ oc create token quay-builder -n virtual-builders --duration
```

Reply:
@SNiemann15  can we add `--duration=20160m` as default for 2weeks. And a note like as can be changed as per requirement.

Reply:
`--duration` needs an argument, something like `--duration 10m`

Reply:
@SNiemann15 it can be with or without `=`,  `--duration 10m` is an example from `oc create token --help`.

`<time_in_minutes>` isn't ideal name, if you run it with `--duration 10`, it'll fail with the message:

```
error: invalid argument "10" for "--duration" flag: time: missing unit in duration "10"
```

I.e. you have to specify units, and it can be seconds, minutes, or hours. For example:

* `--duration=336h`
* `--duration=20160m`
* `--duration=1209600s`
* `--duration=335h60m`

All these options give the same result.

When the option is not specified, the tokens used not to expire. This might have changed in recent OCP versions though.